assets/_scss/core/_utilities.scss

```diff
@@ -16,6 +16,14 @@
     }
 }
 
+// Screen reader only helper
+.sr-only {
```

Comment:
Done in 668c818. 

A little worried bc I couldn't find where I was using that to update there.